pyccel/ast/operators.py

```diff
@@ -447,6 +449,14 @@ class PyccelAdd(PyccelArithmeticOperator):
     """
     _precedence = 12
 
+    def __new__(cls, arg1, arg2):
+        if isinstance(arg1, (LiteralInteger, LiteralFloat)) and \
+           isinstance(arg2, LiteralComplex) and \
+           arg2.real == LiteralFloat(0):
+            return LiteralComplex(arg1, arg2.imag)
```

Comment:
As far as I understand this handles a case like `1 + 2j`, but why shouldn't it also handle `2j + 1`?